NeoMathEngine/src/CMakeLists.txt

```diff
@@ -339,8 +339,9 @@ if(NOT ANDROID AND NOT IOS)
         set(ENV{MKLROOT} ${FINE_ROOT}/ThirdParty/MKL/${CMAKE_SYSTEM_NAME})
     endif()
 
+    set(MKL_USE_STATIC_LIBS ON)
```

Comment:
Isn't it better to use `CACHE BOOL` with description here? Or an `option` with default value `ON`. Now it looks like user can't set this variable to `OFF` outside of this file.